tests/PHPStan/Reflection/Constant/RuntimeConstantReflectionTest.php

```diff
@@ -0,0 +1,47 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Reflection\Constant;
+
+use PhpParser\Node\Name;
+use PHPStan\Testing\PHPStanTestCase;
+use PHPStan\TrinaryLogic;
+
+class RuntimeConstantReflectionTest extends PHPStanTestCase
+{
+
+	public function dataDeprecatedConstants(): array
+	{
+		return [
+			[
+				new Name('\DeprecatedConst\FINE'),
+				TrinaryLogic::createNo(),
+				null,
+			],
+			[
+				new Name('\DeprecatedConst\MY_CONST'),
+				TrinaryLogic::createYes(),
+				null,
+			],
+			[
+				new Name('\DeprecatedConst\MY_CONST2'),
```

Comment:
You should test some deprecated constants from phpstorm-stubs, like `CURLOPT_FTP_SSL`.

Reply:
Also test `FILTER_SANITIZE_STRING` which is deprecated on 8.1+.